Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -25,6 +22,9 @@ public protocol CommentRepliesViewModelInputs {
 
   /// Call when the view loads.
   func viewDidLoad()
+
+  /// Call in `didSelectRow` when either a `ViewMoreRepliesCell` or `CommentViewMoreRepliesFailedCell` is tapped.
+  func viewMoreRepliesOrViewMoreRepliesFailedCellWasTapped()
```

Comment:
This is definitely a long function name but I wanted it to be obvious what was going on here.

Reply:
Updated.